test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthFunctionalTestFixture.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+using Microsoft.Dnx.CommonTestUtils;
+
+namespace Microsoft.Dnx.DesignTimeHost.FunctionalTests.Infrastructure
+{
+    public class DthFunctionalTestFixture : DnxRuntimeFixture
+    {
+        private readonly DisposableDir _context;
+        private readonly int[] _ports;
+
+        private int _nextPortIdx;
+
+        public DthFunctionalTestFixture() : base()
+        {
+            _ports = Enumerable.Range(1774, 20).ToArray();
```

Comment:
Ugh, no use the logic. We have it in other places https://github.com/aspnet/Testing/blob/f4a63f3fca870412d742b181d52aa7e52ec1d434/src/Microsoft.Dnx.TestHost.Client.Sources/TestHostWrapper.cs#L255